include/cinder/Log.h

```diff
@@ -135,18 +140,46 @@ class LoggerBreakpoint : public Logger {
 	Level	mTriggerLevel;
 };
 
-#if defined( CINDER_COCOA )
-
-class LoggerSysLog : public Logger {
+class LoggerSystem : public Logger {
 public:
-	LoggerSysLog();
-	virtual ~LoggerSysLog();
-
-	void write( const Metadata &meta, const std::string &text ) override;
-};
-
+	LoggerSystem();
+	virtual ~LoggerSystem();
+	
+	void write( const Metadata &meta, const std::string &text ) override { mImpl->write( meta, text ); }
+	// SysLog and EventLog logging levels inherently work differently (see specific comments).  Currently
+	// this doesn't matter since logging implimentation only allows once instance of a logger type.
```

Comment:
Ah, yes I see, they are quite different. Essentially what you've done for EventLog is add runtime level filtering support. In the future I'd like to add this at a more generic level, perhaps the LoggerMulti, so we get the functionality for even things like LoggerFile, but I'd like to do it so it is minimal overhead when turned off - that is the tricky part.

It is a tough decision, what to do around ::setlogmask(). Part of me thinks it should be a static LoggerSystem::setLogMask(), that is only available on platforms where syslog exists. If that were done, it would make sense to either move the EventLog's runtime level filtering to LoggerSystem, or remove it altogether, since it will likely come back in a more generic form. 

Any thoughts around the pimpl move? I can get this merged and do the change locally if you prefer, or let me know if you have other opinions. I'd like to get this PR closed soon since it is against glNext, which is going to be removed soon since it was merged into master branch.